pxtblocks/fields/field_tileset.ts

```diff
@@ -314,6 +314,10 @@ export class FieldTileset extends FieldImages implements FieldCustom {
         this.localTile = asset as pxt.Tile;
         super.loadState(pxt.getTSReferenceForAsset(asset));
     }
+
+    getFieldDescription(): string {
+        return lf("tileset");
```

Comment:
i'd make this one "tile" rather than "tileset", since it generally refers to a single tile